### ISSUE TITLE
Lack of possibility to choose location using a screenreader

### ISSUE THREAD

**Issue by Piciok** [Trustroots/trustroots#1116]
It is currently not possible for blind users who rely on screenreaders to make use of any part of the site that requires one to choose the location on a map, E.G. hosting, setting up a meeting offer or looking for hosts.

Steps to reproduce:
1.navigate to any view where choosing a location using the provided map view is required;
2. Try to make the choice using a screenreader such as NVDA or Jaws.
Expected Behaviour: there should be some kind of accessible alternative;
What Happens: as map widgets like this are mostly inaccessible, it is impossible to choose anything.
Desktop
System: Windows 10
Browser: Firefox
but this will be an issue on any platform/browser combination.

Context: blind people access websites and apps on their electronic devices of choice using a screenreader, a special piece of software that presents the text shown on the screen, the controls and the context around them in the form of synthesized speech or braille output on a braille display. This solution works as long as all of the components accessed are coded in an accessible way that is all of the items are able to be recognized by this piece of software and are possible to be navigated through using a keyboard. NVDA, is a good example of a screenreader as it is free and open. Its repo is located at:
https://github.com/nvaccess/nvda
Map views that are essential in order to use all of Trustroot's features are sadly one of those elements that aren't accessible at this time. Because there is no alternative available, a blind person is unable to find hosts or make an offer. My suggestion would be to either add an edit box that accepts geographical coordinates or a Google Maps/Mapquest link from which those can be obtained. Current location could be also used if the device supports GPS.
This minor change would make a huge difference to yet another community of travelers. I hope we can work it out.
